File: abc/abc127/a.java
// https://atcoder.jp/contests/abc127/submissions/43246193

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int b = stdIn.nextInt();

    if(a >= 13) System.out.println(b);
    else if(a >= 6) System.out.println(b / 2);
    else System.out.println(0);
  }
}

File: abc/abc127/a.cpp
// https://atcoder.jp/contests/abc127/submissions/26260590

#include <bits/stdc++.h>
using namespace std;
int main() {
  int a, b;
  cin >> a >> b;
  if(a < 6) cout << 0;
  else if(a < 13) cout << b / 2;
  else cout << b;
}
